exonum/src/blockchain/mod.rs

```diff
@@ -541,6 +546,25 @@ impl Blockchain {
     }
 }
 
+fn service_execute(service: &Service, mut fork: &mut Fork) {
```

Comment:
Do you really need `mut` before `fork` here?

Reply:
Really